app/assets/scss/_iframe_preview.scss

```diff
@@ -43,4 +43,12 @@ $app-mobile-border-bottom-width: 68px;
   @include govuk-media-query($from: tablet) {
       width: 30%;
   }
+
+  h2 span:nth-child(1) {
+    @include govuk-font($size: 24, $weight: bold);
+  }
+
+  h2 span:nth-child(2) {
+    @include govuk-font($size: 19, $weight: bold);
+  }
```

Comment:
What do these do?

Reply:
The preview app stat blocks are a bit custom, so weren't inheriting the page font style. These solve that for that iFrame. I can do some more investigating to work out why it's not inheriting, but thought I'd get it looking right initially.

![Screenshot 2020-09-09 at 15 25 35](https://user-images.githubusercontent.com/22524634/92611472-bdc21300-f2b0-11ea-9eaf-6fe2d261fa3e.png)